scripts/tts_dataset_files/heteronyms-052722

```diff
@@ -0,0 +1,204 @@
+abstract
+abuse
+abuses
+addict
+address
+advocate
+affect
+affiliate
+agape
+aged
+aggregate
+allied
+alternate
+alum
+analyses
+animate
+appropriate
+approximate
+articulate
+associate
+attribute
+august
+axes
+bases
+bass
+blessed
+bologna
+bow
+buffet
+celtic
+close
+combine
+compact
+complex
+compound
+compress
+concert
+concrete
+conduct
+confines
+conflict
+conglomerate
+conjugate
+conscript
+console
+consort
+construct
+consummate
+content
+contest
+contract
+contrast
+converse
+convert
+convict
+coordinate
+correlate
+crooked
+decrease
+defect
+defense
+degenerate
+delegate
+deliberate
+desert
+deviate
+diagnoses
+diffuse
+discard
+discharge
+discount
+do
+document
+does
+dogged
+dove
+duplicate
+elaborate
+ellipses
+entrance
+escort
+estimate
+excuse
+expatriate
+exploit
+export
+expose
+extract
+fragment
+frequent
+graduate
+house
+impact
+implant
+implement
+import
+incense
+incline
+increase
+increment
+initiate
+insert
+instrument
+insult
+interchange
+intimate
+intrigue
+invalid
+invert
+invite
+isolate
+jesus
+job
+laminate
+lead
+learned
+lima
+live
+lives
+mate
+merchandise
+minute
+misuse
+mobile
+moderate
+moped
+mouth
+mow
+multiply
+nestle
+number
+object
+ornament
+overall
+overthrow
+pasty
+perfect
+perfume
+periodic
+permit
+pervert
+pigment
+polish
+postulate
+precipitate
+predicate
+present
+primer
+proceeds
+produce
+progress
+project
+protest
+putting
+ragged
+ravel
+read
+reading
+rebel
+recall
+record
+recount
+recreate
+refund
+refuse
+reject
+rerelease
+resign
+resume
+retard
+retreat
+rodeo
+root
+row
+sake
+second
+separate
+sewer
+shower
+sin
+slough
+sow
+subject
+subordinate
+supplement
+supposed
+suspect
+syndicate
+tarry
+tear
+tier
+transfer
+transform
+transplant
+transport
+unionized
+upset
+use
+uses
+whoop
+wicked
+wind
+winds
+wound
```

Comment:
we should probably add a README along this file about the source of the heteronyms list or comments on top